tests/test_onion_message.py

```diff
@@ -0,0 +1,255 @@
+
+    def test_blinding_privkey(self):
+        a = blinding_privkey(bfh('4141414141414141414141414141414141414141414141414141414141414141'),
+                             bfh('031b84c5567b126440995d3ed5aaba0565d71e1834604819ff9c17f5e9d5dd078f'))
+        self.assertEqual(a, bfh('7e959bf6bdd3a98caf26cbbee7b69678381d5fa2882c6c12eb2042c2367264b0'))
+
+    def test_create_blinded_path(self):
+        pubkey = bfh('02eec7245d6b7d2ccb30380bfbe2a3648cd7a942653f5aa340edcea1f283686619')  # alice
+        session_key = bfh('3030303030303030303030303030303030303030303030303030303030303030')
```

Comment:
any reason why session_key is '303030..' and not '030303...' here?  is this a typo?